src/coreclr/vm/riscv64/cgencpu.h

```diff
@@ -273,7 +273,7 @@ inline BOOL isJump(PCODE pCode)
 
```

Comment:
Are the `isJump` and `isBackToBackJump` methods actually used anywhere? If they are not used, it would be best to delete them.

Reply:
I checked those are not used by searching `isJump` and `isBackToBackJump`.  I just keep because `cgencpu.h` of all archs include the functions. If it is not used at all, I want to remove from all archs in another PR. Thank you.

Reply:
Pushed https://github.com/dotnet/runtime/pull/87357 to remove the methods.

Reply:
Thanks. 

FWIW, the implementation of isJump for RISCV looks incomplete compared to implementation of isJump for other platforms. The implementations for other platforms check each instruction from offset 0 - the RISCV implementation only checks the 3rd instruction. It is not a real problem since it is getting deleted.

Reply:
Thank you so much!